geoserver/webapp/src/docker/Dockerfile

```diff
@@ -28,11 +28,17 @@ RUN mkdir /mnt/geoserver_datadir /mnt/geoserver_geodata /mnt/geoserver_tiles /mn
 
 # add a tweaked configuration. First use case is support of OPTIONS.
 ADD web.xml /var/lib/jetty/webapps/geoserver/WEB-INF/web.xml
+ADD jetty-env.xml /var/lib/jetty/webapps/geoserver/WEB-INF/jetty-env.xml
+# tweaked geowebcache config that use postgresql instead of H2. Copying it to /tmp since copying it 
+# to /mnt/geoserver_tiles won't work (it's a volume, data will be override). File copied to
+# /mnt/geoserver_tiles by a entrypoint script.
```

Comment:
If the image provides `/mnt/geoserver_tiles/geowebcache-diskquota.xml` and `/mnt/geoserver_tiles/geowebcache-diskquota-jdbc.xml`, the first time an empty volume is mounted on it, it shall be initialized with the content provided by the image, which is exactly what we want to achieve here IIUC ?

Reply:
No need to use /tmp then